tests/inspect/inspect.go

```diff
@@ -279,5 +300,77 @@ func main() {
 		fmt.Println("check-cgroups: SUCCESS")
 	}
 
+	if globalFlags.PrintNetNS {
+		ns, err := os.Readlink("/proc/self/ns/net")
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
+		}
+		fmt.Printf("NetNS: %s\n", ns)
+	}
+
+	if globalFlags.PrintIPv4 != "" {
+		iface := globalFlags.PrintIPv4
+		ips, err := test_netutils.GetIPsv4(iface)
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
+		}
+		fmt.Printf("%v IPv4: %s\n", iface, ips[0])
+	}
+
+	if globalFlags.PrintDefaultGWv4 {
+		gw, err := test_netutils.GetDefaultGWv4()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
+		}
+		fmt.Printf("DefaultGWv4: %s\n", gw)
+	}
+
+	if globalFlags.PrintDefaultGWv6 {
+		gw, err := test_netutils.GetDefaultGWv6()
+		if err != nil {
+			fmt.Fprintf(os.Stderr, "%v\n", err)
+			os.Exit(1)
+		}
+		fmt.Printf("DefaultGWv6: %s\n", gw)
+	}
+
+	if globalFlags.PrintGWv4 != "" {
+		iface := globalFlags.PrintGWv4
+		gw, err := test_netutils.GetGWv4(iface)
```

Comment:
So this doesn't work for now because GetGW is not implemented, maybe add a comment here or remove this part until it is implemented to avoid confusion.